code/modules/reagents/reagent_containers/hypospray.dm

```diff
@@ -179,10 +179,38 @@
 	name = "stimpak"
 	desc = "A handheld delivery system for medicine, used to rapidly heal physical damage to the body."
 	icon_state = "stimpakpen"
-	volume = 10
-	amount_per_transfer_from_this = 10
-	list_reagents = list("stimpak" = 10)
-	
+	volume = 15
```

Comment:
This is overall a pretty severe buff for stimpaks, given that they not only heal more but each one comes with more fluid. Ren is requesting a nerf in the form of slower metabolization rate. Are we trying to make them better or worse?

Reply:
15 units, in injections of 5 ren is aware of this and gave me a metabolism number for this.